s3cortxutils/s3cipher/s3cipher/README.md

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
this is md file, '#' will be interpreted differently here. Put the copyright header as plain text, no need of "#' or something else

Reply:
fixed